inspirehep/modules/authors/receivers.py

```diff
@@ -87,11 +87,11 @@ def assign_phonetic_block(sender, *args, **kwargs):
             authors[authors_map[full_name]].update(
                 {"signature_block": signature_block})
 
-        # # For missing phonetic blocks (not valid full names) add None.
+        # For missing phonetic blocks (not valid full names) add None.
```

Comment:
The comment says that you're adding `None`, but you're modifying the code below not to return `None`...